wsl-pro-service/internal/consts/consts.go

```diff
@@ -1,9 +1,16 @@
 // Package consts defines the constants used by the project
 package consts
 
-import log "github.com/sirupsen/logrus"
+import (
+	log "github.com/sirupsen/logrus"
+)
 
 const (
 	// DefaultLogLevel is the default logging level selected without any option.
 	DefaultLogLevel = log.WarnLevel
 )
+
+// Version is the version of the service
+//
+// It is set at build time using the -ldflags option.
+var Version = "Dev"
```

Comment:
We do have ` consts/Version`  and ` common/Version`  then?

It seems that replacing common/Version works for the agent build. Do you know why it doesn’t work in the debian package build? We are in the same condition (the module is outside of the current package). My only bet is that this is due to vendoring maybe? (this is the only different I can see between the two), but can you assert on this before we commit having 2 `Version` variables?

If so, should the other remains in `common/`  as it’s not really common aymore :p

Reply:
I suspect it is due to vendoring, but I'm not 100% sure.

I agree that it makes no sense for the other to be in `common` then.